java/code/src/com/redhat/rhn/frontend/xmlrpc/proxy/ProxyHandler.java

```diff
@@ -32,7 +32,6 @@
 import com.redhat.rhn.frontend.xmlrpc.ProxyNotActivatedException;
 import com.redhat.rhn.frontend.xmlrpc.ProxySystemIsSatelliteException;
 import com.redhat.rhn.frontend.xmlrpc.system.XmlRpcSystemHelper;
-import com.redhat.rhn.manager.entitlement.EntitlementManager;
```

Comment:
I believe that this Java change is correct but I would like to know the reason for it. The commit message is a bit vague about this change.